packages/atomic/src/components/facets-v1/atomic-timeframe-facet/atomic-timeframe-facet.tsx

```diff
@@ -0,0 +1,170 @@
+import {Component, h, State, Prop, VNode, Host} from '@stencil/core';
+import {
+  SearchStatus,
+  SearchStatusState,
+  buildSearchStatus,
+  DateFacet,
+  DateFacetState,
+  buildDateFacet,
+  DateFacetOptions,
+  DateFacetValue,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {FacetPlaceholder} from '../../facets/atomic-facet-placeholder/atomic-facet-placeholder';
+import {FacetContainer} from '../facet-container/facet-container';
+import {FacetHeader} from '../facet-header/facet-header';
+import {FacetValueLink} from '../facet-value-link/facet-value-link';
+import {BaseFacet} from '../facet-common';
+
+/**
+ * A facet is a list of values for a certain field occurring in the results.
+ * An `atomic-timeframe-facet` displays a facet of the results for the current query as date intervals.
+ *
+ * @part facet - The wrapper for the entire facet.
+ * @part placeholder - The placeholder shown before the first search is executed.
+ *
+ * @part label-button - The button that displays the label and allows to expand/collapse the facet.
+ * @part label-button-icon - The label button icon.
+ * @part clear-button - The button that resets the actively selected facet values.
+ * @part clear-button-icon - The clear button icon.
+ *
+ * @part values - The facet values container.
+ * @part value-label - The facet value label, common for all displays.
+ * @part value-count - The facet value count, common for all displays.
+ * @part value-link - The facet value when display is 'link'.
+ */
+@Component({
+  tag: 'atomic-timeframe-facet',
+  styleUrl: 'atomic-timeframe-facet.pcss',
+  shadow: true,
+})
+export class AtomicTimeframeFacet
+  implements InitializableComponent, BaseFacet<DateFacet, DateFacetState> {
+  @InitializeBindings() public bindings!: Bindings;
+  public facet!: DateFacet;
+  public searchStatus!: SearchStatus;
+
+  @BindStateToController('facet')
+  @State()
+  public facetState!: DateFacetState;
+  @BindStateToController('searchStatus')
+  @State()
+  public searchStatusState!: SearchStatusState;
+  @State() public error!: Error;
+  @State() public isCollapsed = false;
+
+  /**
+   * Specifies a unique identifier for the facet.
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The non-localized label for the facet.
+   */
+  @Prop() public label = 'timeframe';
```

Comment:
Do we really need a new label for this ? Just `no title` would do ?

Because we need to remember that we will eventually need to translate all those labels to many different language, so it's annoying if we have a bunch of "only used in one place" labels.

Reply:
It offers a good OOTB experience, but I have nothing against setting notitle or nolabel here again